### PR TITLE
Segment migration

### PR DESCRIPTION
**Pull request by patrickt010**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/guides/segment-migration-guide.md [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+    ```
+    </td>
+    <td>
+    ```typescript
+    setGroup('Working Group', 'UNIVAC');
+    const groupIdentifyObj = new Identify()
+    groupIdentifyObj.set('principles', ['Eckert', 'Mauchly']);
+    groupIdentifyObj.set('site', 'Eckert–Mauchly Computer Corporation');
+    groupIdentifyObj.set('statedGoals', 'Develop the first commercial computer');
+    groupIdentifyObj.set('industry', 'Technology');
```
Comment (justin-fiedler) [inline docs/guides/segment-migration-guide.md:130]: This seeming clunky compared to Segment. I feel chaining the set operations could make it more concise.
```suggestion
    groupIdentifyObj.set('principles', ['Eckert', 'Mauchly']);
     .set('site', 'Eckert–Mauchly Computer Corporation');
     .set('statedGoals', 'Develop the first commercial computer');
     .set('industry', 'Technology');
```

**Review by justin-fiedler — APPROVED** [review]
Looks good 👍

File: docs/guides/segment-migration-guide.md [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+    Documentation for [Browser Typescript SDK](/data/sdks/typescript-browser/).
+
+    <table>
+    <tr>
+    <td>  </td> <td> Segment </td> <td> Amplitude </td>
+    </tr>
+    <tr>
+    <td> Identify </td>
+    <td> 
+    ```typescript
```
Comment (bgiori) [inline docs/guides/segment-migration-guide.md:75]: This is a great workaround for adding code blocks to tables 🤯 

Might be worth documenting somewhere.

File: docs/guides/segment-migration-guide.md [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+| Plugins   | Plugins   | [Plugins](/data/ampli/plugin/) les you extend Amplitude by running a sequence of custom code on every event.|
+
+
+=== "Browser"
+
+    Documentation for [Browser Typescript SDK](/data/sdks/typescript-browser/).
+
+    <table>
+    <tr>
+    <td>  </td> <td> Segment </td> <td> Amplitude </td>
```
Comment (bgiori) [inline docs/guides/segment-migration-guide.md:70]: **Bold** "Segment" and "Amplitude" to match markdown table format.

File: docs/guides/segment-migration-guide.md [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+    </tr>
+    </table>
+
+=== "iOS"
+
+    Documentation for [iOS SDK](/data/sdks/ios/).
+
+    <table>
+    <tr>
+    <td>  </td> <td> Segment </td> <td> Amplitude </td>
```
Comment (bgiori) [inline docs/guides/segment-migration-guide.md:143]: **Bold** "Segment" and "Amplitude" to match markdown table format.

File: docs/guides/segment-migration-guide.md [hunk 5]
```diff
@@ -0,0 +1,300 @@
...
+
+
+
+=== "Android"
+
+    Documentation for [Android Kotlin SDK](/data/sdks/android-kotlin/).
+
+    <table>
+    <tr>
+    <td>  </td> <td> Segment </td> <td> Amplitude </td>
```
Comment (bgiori) [inline docs/guides/segment-migration-guide.md:203]: **Bold** "Segment" and "Amplitude" to match markdown table format.

**Review by bgiori — APPROVED** [review]

File: docs/guides/segment-migration-guide.md (hunk 2 above)
Comment (patrickt010) [reply docs/guides/segment-migration-guide.md:75]: Ha took me a bit to figure this out.

File: docs/guides/segment-migration-guide.md [hunk 6]
```diff
@@ -0,0 +1,300 @@
...
+| write_key | api_key   | Unique key to validate source of the data. |
+| Workspace | Project   | [Projects](https://help.amplitude.com/hc/en-us/articles/360058073772-Create-and-manage-organizations-and-projects#create-a-project) allow you to organize your data.  |
+| User      | User      | User who is performing action.             |
+| Identify  | Identify  | [Identify](/analytics/what-is-amplitude/#user-properties-are-details-about-your-user) updates properties/attributes of the user.|
+| Track     | Event     | [Event](/analytics/apis/http-v2-api-quickstart/) in Amplitude tracks the action user is performing.|
+| Screen    | Event     | Create an Event to track Screen views.|
+| Page      | Event     | Create an Event to track Page views.|
+| Group     | Group     | [Group](/guides/accounts-instrumentation-guide/) is a collection of users. In Amplitude one user could belong to multiple groups. Each group can have properties/attributes that will be available to query/forward on actions performed by any user in the group.|
+| Plugins   | Plugins   | [Plugins](/data/ampli/plugin/) les you extend Amplitude by running a sequence of custom code on every event.|
```
Comment (ppatibandla) [inline docs/guides/segment-migration-guide.md:61]: typo: les -> let's

**Pull request merged**